aten/src/ATen/functorch/BatchRulesLoss.cpp

```diff
@@ -282,6 +303,7 @@ TORCH_LIBRARY_IMPL(aten, FuncTorchBatched, m) {
   m.impl("nll_loss_backward", nll_loss_backward_decomposition);
   m.impl("nll_loss2d_backward", nll_loss_backward_decomposition);
   VMAP_SUPPORT(mse_loss, mse_loss_batch_rule);
+  VMAP_SUPPORT(smooth_l1_loss, smooth_l1_loss_batch_rule);
```

Comment:
minor: Move this below the comment `// mse_loss_backwards uses a decomposition for its batch rule` (as it corresponds to `mse_loss`).

Reply:
Thanks, I will reorder that in an updated commit.